src/main/java/Task.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Encapsulates the Task object.
+ * The Task object includes its own description
+ * as well as if any of the object instantiated is done or uncompleted.
+ * Also, it keep tracks of a counter of all Task objects that are instantiated.
+ * The Task object provides the extension to other children classes such as Todo, Deadline and Event.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    String typeOfTask = "";
```

Comment:
Why is this attribute given a dummy value here, while the two attributes above are not?

Reply:
Yes, it is redundant. I have removed it. Thanks.